docs/resource/host.md

```diff
@@ -6,6 +6,8 @@
 
 **Description:** A host is defined as a computing instance. For example, physical servers, virtual machines, switches or disk array.
 
+Note that `host.*` namespace should only be used for resource attributes. For hosts' metrics the `system.*` namespace should be used.
```

Comment:
```suggestion
The `host.*` namespace should be exclusively used to capture resource attributes. To report host metrics, use the `system.*` namespace should be used.
```

Maybe? Also, is this supposed to be a normative text or not? Should "should" be "SHOULD" or "MUST"?